src/app/home/home.component.html

```diff
@@ -16,103 +16,209 @@
-                <a mat-raised-button color="accent" routerLink="onboarding">Setup Account <mat-icon>chevron_right</mat-icon></a>
-              </div>
-              <div class="col-sm-6 col-xs-12 col-md-12">
-                <a mat-raised-button color="accent" href="https://docs.dockstore.org/news/">News and Events <mat-icon>chevron_right</mat-icon></a>
-              </div>
-              <div class="col-sm-6 col-xs-12 col-md-12" style="margin-bottom: 20px">
-                <a mat-raised-button color="accent" href="https://discuss.dockstore.org/">Discuss <mat-icon>chevron_right</mat-icon></a>
-              </div>
-            </div>
-        </div>
+    </div>
+    <div fxFlex>
+      <div fxLayout="column">
+        <mat-icon fxFlexAlign="center" style="font-size: 50px; height: 50px">cloud</mat-icon>
+        <h3 class="text-center">Cloud launch</h3>
+        <p>Users can launch tools and workflows in a variety of cloud platforms.</p>
       </div>
     </div>
   </div>
 </div>
-<div class="container mainScreen" #mainScreen>
-  <div>
-    <div class="col-md-9 col-sm-12 home-list">
-        <tabset class="homeComponent" type="tabs">
-          <tab id="toolTab" [customClass]="browseToolsTab" (select)="onSelect($event)">
-            <ng-template tabHeading>
-              <img class="site-icons-tab" src="assets/images/dockstore/dockstore-tools-blue.png">
-              Browse Tools
-            </ng-template>
-            <p>A tool is a docker container with an associated descriptor describing how to run it.</p>
-            <span *ngIf="selectedTab === 'toolTab'">
-                <app-list-containers [previewMode]="true"></app-list-containers>
-            </span>
-          </tab>
-          <tab id="workflowTab" [customClass]="browseWorkflowsTab" (select)="onSelect($event)">
-            <ng-template tabHeading>
-              <img class="site-icons-tab" src="assets/images/dockstore/dockstore-workflows-green.png">
-              Browse Workflows
-            </ng-template>
-            <p> A workflow is a series of tools strung together, with an associated descriptor describing how to run it.</p>
-            <span *ngIf="selectedTab === 'workflowTab'" class="m-2">
-              <app-list-workflows [previewMode]="true"></app-list-workflows>
-            </span>
-          </tab>
-        </tabset>
+
+<div class="container stripe-holder">
+  <div fxFlex fxLayout="column" fxLayoutGap="20px">
+    <h3 class="text-center">Share your tools now</h3>
+
+    <div fxLayout="row wrap" fxLayoutAlign="center center" fxLayoutGap="20px">
+      <a mat-stroked-button href="https://docs.dockstore.org/docs/prereqs" target="_blank" rel="noopener noreferrer">
+        Getting Started Guide
+      </a>
+      <button mat-stroked-button routerLink="/organizations">
+        Discover Organizations
+      </button>
+      <button mat-stroked-button routerLink="/search">
+        Discover Tools and Workflows
```

Comment:
When you click both discover buttons (here and in the scientist section), the new page opens up scrolled to the bottom.